### PR TITLE
{bio}[foss/2021b] Biopython v1.79 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Biopython/Biopython-1.79-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ bioinformatics. """
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://biopython.org/DIST']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['edb07eac99d3b8abd7ba56ff4bedec9263f76dfc3c3f450e7d2e2bcdecf8559b']
+
+dependencies = [
+    ('Python', '3.9.6', '-bare'),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/Biopython/Biopython-1.79-foss-2021b.eb:20]: @sassy-crick the `-bare` python can make sense as a builddep, but usually not as a dependency and certainly not together with `SciPy-bundle`...

Comment (sassy-crick) [reply easybuild/easyconfigs/b/Biopython/Biopython-1.79-foss-2021b.eb]: @migueldiascosta That makes sense. I changed that. Thanks!

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**